lib/router/router.js

```diff
@@ -15,6 +19,10 @@ export default class Router extends EventEmitter {
 
     // set up the component cache (by route keys)
     this.components = { [this.route]: { Component, err } }
+    // contain a map of response of prefetched routes
+    this.prefetchedRoutes = {}
+    this.prefetchingLockManager = new LockManager(2)
+    this.prefetchingRoutes = {}
```

Comment:
@nkzawa even with it or without it, since we are fetching parallel we've to do it.
In the previous version, we hadn't that check. So, it was possible to fetch the same page multiple times.